source/contact.erb

```diff
@@ -0,0 +1,104 @@
+    address: |
+      303 Buchanan Chambers
+      Cnr Pearce & Warwick Street
+      7735
+      Claremont, SA
+    humanized_phone_number: "+27 21 671 4503"
+    phone_number: "+27216714503"
+    email: "hello@unboxed.co"
+    map_link: "http://goo.gl/maps/JkNC9"
+    map_image: "https://a.tiles.mapbox.com/v4/web-unboxed.oec60hbb/attribution,zoompan,zoomwheel,geocoder,share.html?access_token=pk.eyJ1Ijoid2ViLXVuYm94ZWQiLCJhIjoiY2lpN2hneWlvMDA4YnYzbHg2NHFqbW9yaSJ9.J1ksUtXP8Xsb2piQUcWJfA"
+
+    transport_directions: "nearest parking and station"
+    station_and_parking:
+      - name: "grove avenue"
+        link: "https://www.parkme.com/map?lot=122336"
+        icon: "transport-icon__car.png"
+      - name: "claremont"
+        link: "http://www.capemetrorail.co.za/Route-Map.html"
+        icon: "transport-icon__train.png"
+---
+
+<header class="page-header">
+  <div class="page-header__container">
+    <h1 class="page-header__title">
+      <%= current_page.data.title %>
+    </h1>
+
+    <p class="page-header__subtitle">
+      <%= current_page.data.subtitle %>
+    </p>
+  </div>
+</header>
+
+<div class="container--contact-page">
+  <% current_page.data.offices.each do |office| %>
+    <div class="contact-page-tile">
+      <div class="contact-page-tile__container">
+        <div class="contact-page-tile__content">
+          <p class="contact-page-tile__title"><%= office.name %></p>
+          <address class="contact-page-address">
+            <%= simple_format office.address %>
+          </address>
+          <div class="contact-page-address__phone-number">
+            <a href="tel:<%= office.phone_number %>">
+            <i class="fa fa-phone contact-page-address__phone-icon"></i>
+              <%= office.humanized_phone_number %>
+            </a>
+          </div>
+          <div class="contact-page-address__email"><a href="mailto:<%= office.email %>">
+            <i class="fa fa-envelope contact-page-address__email-icon"></i>
+            <%= office.email %>
+          </a></div>
+          <div class="contact-page-address__map-link">
+            <a href="<%= office.map_link %>" target="_blank">
+            <i class="fa fa-map-marker fa-lg contact-page-address__map-icon"></i>
+              open in maps
+            </a>
+          </div>
+          <div class="contact-page-address__directions">
+            <%= office.transport_directions %>
+            <% office.station_and_parking.each do |place| %>
+             <p class="contact-page-address__directions-station">
+               <a href="<%= place.link %>" target="_blank">
+                 <%= image_tag place.icon, class: 'contact-page-address__transport-icon' %>
+                 <%= place.name %>
+               </a>
+             </p>
+            <% end %>
+          </div>
+        </div>
+        <div class="contact-page-tile__image-container">
+          <iframe width='100%' height='500px' frameBorder='0' src="<%= office.map_image %>"></iframe>
```

Comment:
Does this really need a container element? It seems the only reason we have it in is to then have a selector for the iframe within it:

`.contact-page-tile__image-container iframe`

We can remove the div and just use the iframe and add the following class instead:

`.contact-page-tile__image`

However, we probably also want to add `display: block` to the element to remove the additional padding that inline elements have by default (see the small change at the bottom white spacing):

Current:
![screen shot 2015-12-18 at 12 18 52](https://cloud.githubusercontent.com/assets/885223/11896839/d6a9cab0-a581-11e5-8741-7ec3abc75b71.png)

With `display: block;`
![screen shot 2015-12-18 at 12 19 01](https://cloud.githubusercontent.com/assets/885223/11896840/d6b35a9e-a581-11e5-864a-c66250822fb6.png)